dts/arm64/ti/ti_am6232_a53.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Enphase Energy
+ * Copyright (c) 2025 Ayush Singh, BeagleBoard.org Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm64/ti/ti_am62x_a53.dtsi>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@1 {
```

Comment:
How about we add all for cores to `ti_am62x_a53` and set them to `disabled`? This would make this and the am6234 file obsolete and each board can decide which core to enable.